### ISSUE TITLE
Add Zone default ttl

### ISSUE THREAD

**Issue by adriancole** [Netflix/denominator#357]
Zone.ttl could reflect the default TTL value for all records created. This could use provider-specific functionality or instead lookup the SOAData.minimum and apply it.

This was attempted in #351, but not practical due to lack of support.
 * route53 - unsupported
 * ultradns - unsupported
 * google cloud dns - unsupported
 * dynect - supported on zone create, but don't know how to view or update
 * rackspace, clouddns - supported on zone create and update (likely due to BIND)

Using the SOAData.minimum in lieu of provider support would make this feature expensive to implement, as that would imply we need to lookup or cache the SOA for any RRSet update that set the ttl to null.

**Comment by adriancole** [comment]
One lighter weight means to do zone default ttl could be to just expose a way to tell denominator what ttl to use when one isn't set in the rrset. This would eliminate a network call dependency and also the non-portable dilemma.

**Comment by adriancole** [comment]
In DynECT SOA.ttl controls the default ttl of new records

**Comment by jdamick** [comment]
for ultradns you can set the default TTL per record type in the SOAP API, see: saveAccountPreferences

**Comment by adriancole** [comment]
@jdamick so, in principle, I'd be able to eliminate the TTL attribute from the below requests, then?

```
createResourceRecord
addDirectionalPoolRecord
updateDirectionalPoolRecord
addRecordToRRPool
updateRecordOfRRPool
```

**Comment by adriancole** [comment]
I don't think we will be able to easily make a default zone ttl portable. However, we shouldn't throw the baby out with the bathwater.

Let's make it possible to display the zone (SOA) ttl conceding it may not imply anything else.

We should go back and correct rrset.ttl docs. They say it falls back to the default zone ttl which isn't entirely true. As we found in this thread, a default could be from the account, zone, or even somewhere else (like hard coding :))

**Comment by jdamick** [comment]
sounds good, i need to check the new REST api to see where that stands as well.

**Comment by adriancole** [comment]
Cleared out comments after reworking title and description
